handlers/high-charts/src/utils.ts

```diff
@@ -88,3 +88,35 @@ export async function sendOSC(jsonFile: string, outFile: string, server: string,
         })
     ]);
 }
+
+/**
+ * The function will return the string representing the graph title and axes information
+ * @param highChartsData 
+ * @returns graphInfo string
+ */
+export function getGraphInfo(highChartsData: any): string{
+    const chartsData = JSON.parse(JSON.stringify(highChartsData));
```

Comment:
Is this actually necessary? We aren't writing to chartsData so I don't think we need to worry about overwriting the source.